types/trigger.d.ts

```diff
@@ -140,7 +149,7 @@ export type TriggerSet<Data extends RaidbossData> = {
   overrideTimelineFile?: boolean;
   timelineFile?: string;
   timeline?: TimelineFunc;
-  triggers?: NetRegexTrigger<Data>[];
+  triggers?: (NetRegexTrigger<Data> | RegexTrigger<Data>)[];
```

Comment:
I think this should stay `NetRegexTrigger<Data>` honestly.  If somebody is itching to convert a file that has a regex hp trigger in it, they can convert it to the new utility first.  I think it's fine to be stricter about the kinds of triggers that exist in built-in cactbot.

Reply:
This causes issues downstream in PopupText that would need some coercion to fix, IIRC, though I'm not at my desktop now to confirm.